docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -804,6 +804,79 @@ Smithy will generate the following OpenAPI model:
         ]
     }
 
+----------------
+``@examples`` trait conversion
+----------------
+
+In Smithy, example values of input structure members and the corresponding
+output or error structure members for an operation are grouped together
+into one set of example values for an operation. Below is an example "unit" of ``FooOperation``
+operation shape, which shows this logical grouping.
+
+.. code-block:: smithy
+
+    apply FooOperation @examples(
+        [
+            {
+                title: "valid example",
+                documentation: "valid example doc",
+                input: {
+                    bar: "1234"
+                },
+                output: {
+                    baz: "5678"
+                },
+            }
+        ]
+    )
+
+However, example values in OpenAPI are scattered throughout the model, with each example value
```

Comment:
Oh interesting. It's unfortunate that you can't provide a fully example of a request's input (headers + body) or response headers + body. You have to break them up into isolated examples that don't show full messages.